Add full name for cantons

eduardocorrearaujo opened the pull request.

eduardocorrearaujo committed:
Add full name for the title of the figures

eduardocorrearaujo committed:
Merge branch 'review_changes' of https://github.com/thegraphnetwork/COVID-CH-dashboard into review_changes

eduardocorrearaujo committed:
Add full name for the canton and scatter plots

fccoelho commented on for_other_cantons.py:
```diff
@@ -294,28 +310,55 @@ def plot_forecast_canton(table_name,canton, curve, title= None):
     
     st.write(f'''
-             The graphs below show the number of cases and hospitalizations in {canton}
+             The graphs below show the number of cases and hospitalizations in Geneva
```
Here the string should be
```python
f"... hospitalizations in {full_name_canton}"
```

fccoelho commented on forecast.py:
```diff
@@ -388,7 +471,7 @@ def download_button(object_to_download, download_filename, button_text, pickle_i
 
 def app(): 
     
-    st.title('Switzerland COVID-19 Hospitalizatiokn forecasts')
```
Esta linha já está corrigida no ramo main: https://github.com/thegraphnetwork/COVID-CH-dashboard/blob/0b3acc9e4daec8b56d2007f2b02d0394b389a0b7/forecast.py#L391

eduardocorrearaujo committed:
Merge branch 'main' into review_changes

fccoelho commented on get_data.py:
```diff
@@ -19,6 +19,15 @@
+    return df
+
 
 def build_lagged_features(dt, maxlag=2, dropna=True):
```
Are these functions really necessary now that model training is done outside of the dashboard?

eduardocorrearaujo replied:
I just removed the unnecessary functions.

eduardocorrearaujo committed:
Minor changes

- I make the changes requested in the issue
- I change the position of the cases vs hospitalizations scatter plot for Switzerland.
